tests/integration_tests/async_events/api_tests.py

```diff
@@ -33,6 +33,7 @@ def fetch_events(self, last_id: Optional[str] = None):
 
     @mock.patch("uuid.uuid4", return_value=UUID)
     def test_events(self, mock_uuid4):
+        app._got_first_request = False
```

Comment:
a bit of an hack, `async_query_manager.init_app(app)` registers a Flask `app.before_request`. New Flask will only allow registering before request hooks if not request was yet processed by the app.